service/kube_slice_resource_names.go

```diff
@@ -184,6 +185,11 @@ var (
 			APIGroups: []string{apiGroupKubeSliceWorker},
 			Resources: []string{resourceWorkerSliceConfig, resourceWorkerSliceGateways, resourceWorkerServiceImport},
 		},
+		{
```

Comment:
Add same verbs in status as well if you need.

Reply:
Also, if UI needs to perform actions on the CR, then add rules in `readOnlyRoleRules` and `readWriteRoleRules` as well.